controllers/k8ssandra/medusa_reconciler.go

```diff
@@ -130,7 +131,12 @@ func (r *K8ssandraClusterReconciler) reconcileMedusa(
 			return result.RequeueSoon(r.DefaultDelay)
 		}
 		// Create a cron job to purge Medusa backups
-		purgeCronJob, err := medusa.PurgeCronJob(dcConfig, kc.SanitizedName(), namespace, logger)
+		operatorNamespace, err := os.ReadFile("/var/run/secrets/kubernetes.io/serviceaccount/namespace")
```

Comment:
This doesn't work in envtests.

Reply:
**Question:** Didn't we agree on using an env variable to make the namespace available at runtime, while being envtests friendly?

Reply:
Yes, I haven't pushed a change yet. I'm having trouble with envtests picking up the variable as it's not setup for the tests.